### DISCUSSION TITLE
Container Registry is deprecated and scheduled for shutdown.  After March 18, 2025, Container Registry will be shut down.

### DISCUSSION THREAD

**Discussion by julien-michaud** [aquasecurity/trivy#8002] [Adopters]
see https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr

`gcr.io` is now used to pull the db images https://github.com/aquasecurity/trivy/issues/7938

We might have a problem in the near futur 😅

**Comment by knqyf263** [comment]
The document says

![CleanShot 2024-11-27 at 19 32 06](https://github.com/user-attachments/assets/23bc2208-5326-46a3-a3de-569b57421d32)

**Comment by nikpivkin** [comment]
From the table of differences between container registry and artefact registry:
> mirror.gcr.io is a [pull-through cache](https://cloud.google.com/container-registry/docs/pulling-cached-images) that stores the most frequently requested Docker Hub images across all users. mirror.gcr.io is in now hosted on Artifact Registry.

**Comment by simar7** [comment]
I don't think so as mirror.gcr.io is Artifact Registry not Container Registry (which is being deprecated).
